source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,153 @@
+No.  Control                                                Function
+===  =====================================================  ============================================================================================
+1    :hwlabel:`TRIM` knob                                   Adjust inidvidual channel output gain
+2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs      Adjust high/mid/low-frequencies
+3    :hwlabel:`FILTER` knob                                 Low pass / high pass filter for each channel
+4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
+4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
+5    Channel faders                                         Adjust the output level for each channel
+6    Cross fader                                            Fade between left and right deck
+7    :hwlabel:`MASTER` knob                                 Adjust master output level
+8    :hwlabel:`HEADPHONES` knob                             Adjust headphones output level
+9    :hwlabel:`MASTER` button                               Output master to headphones
+===  =====================================================  ============================================================================================
+
+Performance Pads
+~~~~~~~~~~~~~~~~
+
+You can use the Pad Mode Select buttons to select a mode for the
+performance pads. Performance pads are numbered from top left to bottom right. First row is 1-4 and the second row is 5-8.
+
+Hot Cue Mode
+^^^^^^^^^^^^
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_hotcue.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (performance pads)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-SB3 (performance pads)
+
+Bookmark positions in the track and jump to them.
+
+========  ===============================================================  ==========================================
+No.       Control                                                          Function
+========  ===============================================================  ==========================================
+1-4       Pad (unlit)                                                      Save current position as hot cue.
+1-4       Pad (lit)                                                        Jump to hot cue.
+1-4       :hwlabel:`SHIFT` + Pad (lit)                                     Clear hot cue.
+5         Pad                                                              Previous track
+6         Pad                                                              Search left
+7         Pad                                                              Search right
+8         Pad                                                              Censor
```

Comment:
Censor would be more intuitive if it was on SHIFT±PLAY.

Reply:
These are the original (OOB) mappings for the controller.